src/components/manage/Blocks/Accordion/View.jsx

```diff
@@ -181,15 +189,15 @@ const View = (props) => {
                 <AnimateHeight
                   animateOpacity
                   duration={500}
-                  height={active ? 'auto' : 0}
+                  height={active || detectIfDiff() ? 'auto' : 0}
```

Comment:
@dobri1408 Wouldn't it be more efficient to store the output of `detectIfDiff()` into a `const` and then use it here and later in the code?